src/App.js

```diff
@@ -1,6 +1,6 @@
 import LandingPage from './Pages/LandingPage';
 import 'antd/dist/antd.css'; // or 'antd/dist/antd.less'
-import NavBar from './Components/NavBar';
```

Comment:
Is this component still being used anymore?  If not lets remove it along with any css for it from the code base.

Reply:
Scratch that, I see now that the file was deleted.

Reply:
I will note that the index part of the import is dedundant, but not incorrect.